src/components/new-request/business-lookup.vue

```diff
@@ -43,6 +42,26 @@
           Ensure you have entered the correct business name or number.
         </p>
       </template>
+      <!-- Customize the results template -->
+      <template #item="{ item }">
+        <v-row class="business-lookup-result pt-1">
+          <v-col cols="3">
+            <div class="result-identifier">
+              {{ item.identifier }}
+            </div>
+          </v-col>
```

Comment:
This looks great! I think you can probably get rid of the div by putting the class on the v-col element, which would simplify this a bit.

Reply:
What about the other 2? 😁

Reply:
Done 👍